alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -202,14 +202,26 @@ class AlertEngineImplTest
         |    - test@phdata.io
         |  """.stripMargin
 
-    val app1 = AlertEngineConfigParser.convert(yaml).applications(0)
-    val app2 = AlertEngineConfigParser.convert(yaml).applications(1)
-
     val triggeredAlerts =
       List(
-        (app1, Option(TriggeredAlert(app1.alertRules(0), "spark1", null, 1))),
-        (app2, Option(TriggeredAlert(app2.alertRules(0), "spark2", null, 2))),
-        (app2, Option(TriggeredAlert(app2.alertRules(1), "spark2", null, 2)))
+        (AlertEngineConfigParser.convert(yaml).applications(0),
```

Comment:
Looks like formatting just changed here? We are using a tool called 'scalafmt' (https://scalameta.org/scalafmt/) I is an sbt (and intellij) plugin. The sbt compile step will format the code. Make sure you run `sbt compile` before you check in you code for final review.